packages/core/admin/admin/src/pages/Admin/Onboarding/index.js

```diff
@@ -98,37 +93,113 @@ const Onboarding = () => {
+                alignItems="end"
+                justifyContent="space-between"
+                paddingBottom={4}
+                paddingRight={5}
+                paddingLeft={5}
+                paddingTop={5}
               >
-                <Stack horizontal spacing={2}>
-                  {link.Icon}
-                  <Typography color="neutral600">{link.label}</Typography>
-                </Stack>
-              </StyledLink>
-            ))}
-          </LinksWrapper>
-        </FocusTrap>
+                <Typography fontWeight="bold">
+                  {formatMessage({
+                    id: 'app.components.Onboarding.title',
+                    defaultMessage: 'Get started videos',
+                  })}
+                </Typography>
+                <TextLink
+                  as="a"
+                  href="https://www.youtube.com/playlist?list=PL7Q0DQYATmvidz6lEmwE5nIcOAYagxWqq"
+                  target="_blank"
+                  rel="noreferrer noopener"
+                  variant="pi"
+                  textColor="primary600"
+                >
+                  {formatMessage({
+                    id: 'app.components.Onboarding.link.more-videos',
+                    defaultMessage: 'Watch more videos',
+                  })}
+                </TextLink>
+              </Flex>
+              <Divider />
+              {VIDEO_LINKS.map(({ href, duration, label }, index) => (
+                <VideoLinkWrapper
+                  as="a"
+                  href={href}
+                  target="_blank"
+                  rel="noreferrer noopener"
+                  key={href}
+                  hasRadius
+                  paddingTop={4}
+                  paddingBottom={4}
+                  paddingLeft={6}
+                  paddingRight={11}
+                >
+                  <Box paddingRight={5}>
+                    <Typography textColor="neutral200" variant="alpha">
+                      {index + 1}
+                    </Typography>
+                  </Box>
+                  <Box position="relative">
+                    <Preview src={OnboardingPreview} alt="onboarding preview" />
+                    <IconWrapper
+                      position="absolute"
+                      top="50%"
+                      left="50%"
+                      background="primary600"
+                      borderRadius="50%"
+                      justifyContent="center"
+                      width={6}
+                      height={6}
+                    >
+                      <Icon as={Play} color="buttonNeutral0" width={3} height={3} />
+                    </IconWrapper>
+                  </Box>
+                  <Flex direction="column" alignItems="start" paddingLeft={4}>
+                    <Typography fontWeight="bold">{formatMessage(label)}</Typography>
+                    <Typography textColor="neutral600" variant="pi">
+                      {duration}
+                    </Typography>
```

Comment:
You could add a `VisuallyHidden` text in between with `:` which will make screenreaders pause when they read the title and the duration.

Reply:
oh I didn't know that thanks for the tip
to make sure we're aligned, something like this:
```js
<Typography textColor="neutral600" variant="pi">
  {minutes}:<VisuallyHidden>:</VisuallyHidden>
  {seconds}
</Typography>
```

Reply:
Not quite. More like:

```js
<Typography fontWeight="bold">
  {formatMessage(label)}
  <VisuallyHidden>:</VisuallyHidden>
</Typography>

<Typography textColor="neutral600" variant="pi">
  {duration}
</Typography>
```

Reply:
ohh that makes sense, pause between label and duration 🙏